src/lib.rs

```diff
@@ -1431,6 +1437,16 @@ impl Builder {
         self.options.wasm_import_module_name = Some(import_name.into());
         self
     }
+
+    /// Specify the dynamic library name if we are generating bindings for a shared library.
+    pub fn dynamic_library_name<T: Into<String>>(
+        mut self,
+        dynamic_library_name: T,
+    ) -> Self {
+        self.options.dynamic_loading = true;
```

Comment:
Do you really need to keep track of `self.options.dynamic_loading`? It seems you really just need a library name, so the `dynamic_loading` checks could just be `self.options.dynamic_library_name.is_some()`.

Reply:
Fixed this.